src/commands/modelReferences.ts

```diff
@@ -0,0 +1,157 @@
+import { promises as fs } from 'fs';
+import * as path from 'path';
+import * as vscode from 'vscode';
+import { XMLParser } from 'fast-xml-parser';
+import { runXMLExporter } from '../tla2tools';
+
+export interface ModuleReferences {
+    extends?: Set<string>;
+    instances?: Set<string>;
+}
+
+/**
+    Extracts module references from a model file.
+    - EXTENDS clauses are read from the source text to avoid false positives when SANY
```

Comment:
I don't see a reason why SANY's XML exporter cannot include this information.  Let's get this feature added instead of relying on parsing the file.